enhancements/template.md

```diff
@@ -193,6 +193,22 @@ enhancement:
 - What changes (in invocations, configurations, API use, etc.) is an existing
   cluster required to make on upgrade in order to make use of the enhancement?
 
+Upgrade expectations:
+- You only need to support `N->N+1` upgrade steps - so, for example, it is
+  acceptable to require a user running 4.3 to upgrade to 4.5 with a `4.3->4.4`
+  step followed by a `4.4->4.5` step.
+- You may postpone some upgrade migrations and set `Upgradeable:False` until they
+  are completed, if they are required before another upgrade.
```

Comment:
Though this is discouraged, we should not be asking for human intervention and should be operating the platform ourselves if at all possible.

Reply:
also clarify "before another minor upgrade"

Reply:
This sentence is really confusing. Mostly because "they" has no antecedent ("tasks your operator needs to perform before upgrading"?) but also because it doesn't even say that it's talking about operators.

Reply:
My understanding is this is about migration tasks that an operator completes after it has reported an upgrade has completed, and the operator blocks other upgrades from happening until after the operator has completed those tasks.

I was trying to capture Clayton's comment elsewhere:

> there are certain migrations that happen only after we reach a version
￼> the apiserver team has the only example of that today - which is after we reach level of 4.4 they can go do the "things that must be done before 4.5 can be upgraded" and set upgradeable=false until that completes

I think I'll just drop this, since I don't have time right now to fully dig and understand this guidance.